build/certificates/README.md

```diff
@@ -0,0 +1,105 @@
+# IBM FHIR Server - Development Certificates 
+
+The certificates are used in development and CI/CD.
+These certificates are not configured for production use.
+
+The sections in this document walks through creation of the certificates in a temporary folder, and then subsequent update inline.  
+
+The Audit Kafka jks are updated separately. 
+
+# Create CA and Certs and P12s (does not replace)
+
+This script should be run first, and subsequently each many section must update their cert/key. 
+
+1. Set `export CHANGE_PASSWORD=change-password`
+
+2. Set `export WORKSPACE=$(pwd)`
+
+3. Run `bash ${WORKSPACE}/build/certificates/create-ca-and-certs.sh` to update the certs.
+
+At the end the tmp folder contains a copy of the certs and keys with a CA Issuer of `Issuer: C=US, O=IBM Corporation, OU=IBM Watson Health, OU=FHIR`.
+
+# Minio: Continuous Integration Certificates
+
+The following certifiates are used with minio. 
```

Comment:
Misspelling "certifiates" in multiple places in this file.

Reply:
fixed